optimizely/config/datafileProjectConfig/mappers/condition_trees_test.go

```diff
@@ -60,6 +61,45 @@ func TestBuildAudienceConditionTreeSimpleAudienceCondition(t *testing.T) {
 	assert.Equal(t, expectedConditionTree, conditionTree)
 }
 
+func TestBuildConditionTreeUsingDatafileAudienceConditions(t *testing.T) {
+	datafile, err := resources.GetTestDataFileJSON("bucketing_id.json")
```

Comment:
We shouldn't need to load a full datafile for this. Instead just construct the `datafile.Audiences` array as a templated string like: 

```
datafileAudience = datafile.Audience{
  Conditions...
}
```

Reply:
Recommended changes made.